api/src/main/java/me/lokka30/treasury/api/economy/currency/Currency.java

```diff
@@ -94,7 +95,7 @@ public interface Currency {
      * @return boolean value
      * @since v2.0.0
      */
-    boolean supportsNegativeBalances();
+    TriState supportsNegativeBalances();
```

Comment:
Was hoping to revert this to a boolean because ambiguous responses don't really make sense, but it's probably fine as-is either way. Definitely a minor nitpick.

Reply:
Agreed, this is a yes or no response.